src/fava/core/ingest.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, ledger):
         super().__init__(ledger)
         self.config = []
         self.importers = {}
+        self.hooks = [extract.find_duplicate_entries]
```

Comment:
I think it would be better to initialise to an empty list here and and then set it to the default below.